src/app/functions/server/featureFlags.ts

```diff
@@ -7,10 +7,22 @@ import {
   FeatureFlag,
   getFeatureFlagByName,
 } from "../../../db/tables/featureFlags";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../api/utils/auth";
+
+export type FeatureFlagsEnabled = {
+  FreeBrokerScan: boolean;
+  PremiumBrokerRemoval: boolean;
+};
+
+export async function isFlagEnabled(name: string): Promise<boolean> {
```

Comment:
You can make this same change for every function inside `tables/featureFlags.ts`.

```suggestion
export async function isFlagEnabled(name: keyof FeatureFlagsEnabled): Promise<boolean> {
```